tests/network/primary_pod_network.go

```diff
@@ -235,6 +239,14 @@ func newFedoraWithGuestAgentAndDefaultInterface(iface v1.Interface) (*v1.Virtual
 	return vmi, nil
 }
 
+func removeCIDRs(ips []string) []string {
+	ipAddresses := []string{}
+	for _, ip := range ips {
+		ipAddresses = append(ipAddresses, removeCIDR(ip))
+	}
+	return ipAddresses
+}
+
 func removeCIDR(ip string) string {
```

Comment:
nit: Calling this from the caller could just be is simply `removeCIDRs([]string{ip})` and then we can have one way of doing something and not multiple ways.
But I guess we can improve it in the future, as I see this is happening way too often.

Reply:
We can't the `WithTransform` expect the correct function firm.

Reply:
Right, we need to fix it from the root.